index.html

```diff
@@ -10312,127 +10330,239 @@ <h2>User prompts</h2>
  If the subsequent requested <a>command</a> is not one listed in this chapter,
  an <a>unexpected alert open</a> <a>error</a> will be returned.
 
-<p><a>User prompts</a> that are spawned
- from <a><code>beforeunload</code></a> event handlers,
- are <a>accepted</a> implicitly upon <a>navigation</a>
- or <a>close window</a>,
- regardless of the defined <a>user prompt handler</a>.
+<p>Whenever <a>active sessions</a> is a list containing exactly one
+item, and that item is a <a>HTTP session</a>, but is not a <a>BiDi
+session</a>, then in the <a>steps to fire beforeunload</a>,
+implementations must act as if showing an unload prompt is likely to
+be annoying, deceptive, or pointless.
+
+<p class=note>This means that beforeunload prompts are never shown
+when there&apos;s an active HTTP-only session.
 
 <p>A <a>user prompt</a> has an associated <dfn>user prompt message</dfn>
  that is the string message shown to the user,
  or <a><code>null</code></a> if the message length is <code>0</code>.
 
-<p>The following <dfn>table of simple dialogs</dfn>
- enumerates all supported <a>simple dialogs</a>,
- along with the <a>commands</a> that are allowed to interact with it
- as a non-normative reference:
+<p>The <dfn>current user prompt</dfn> is the <a>user prompt</a> which
```

Comment:
How does it work with multiple prompts showing at the same time? I guess it is a pre-existing behavior, but does it mean that the second prompt (a different browsing context) will override the first one?

Reply:
I've fixed this up to try to formalize the fact that we only check for prompts in the current browsing context. Obviously that means we can't use this concept in BiDi, but I think that's OK because we don't need it.